go/vt/schemadiff/capability_test.go

```diff
@@ -0,0 +1,195 @@
+			create:                    "create table t(id int, i1 int not null, primary key(id))",
+			alter:                     "alter table t modify column i1 bigint not null default 3",
+			expectCapableOfInstantDDL: false,
+		},
+		{
+			name:                      "set column dfault value to null",
+			create:                    "create table t(id int, i1 int, primary key(id))",
+			alter:                     "alter table t modify column i1 int default null",
+			expectCapableOfInstantDDL: true,
+		},
+		// enum/set:
+		{
+			name:                      "change enum default value",
+			create:                    "create table t(id int, c1 enum('a', 'b', 'c'), primary key(id))",
+			alter:                     "alter table t modify column c1 enum('a', 'b', 'c') default 'b'",
+			expectCapableOfInstantDDL: true,
+		},
+		{
+			name:                      "enum append",
+			create:                    "create table t(id int, c1 enum('a', 'b', 'c'), primary key(id))",
+			alter:                     "alter table t modify column c1 enum('a', 'b', 'c', 'd')",
+			expectCapableOfInstantDDL: true,
+		},
+		{
+			name:                      "enum append with changed default",
+			create:                    "create table t(id int, c1 enum('a', 'b', 'c') default 'a', primary key(id))",
+			alter:                     "alter table t modify column c1 enum('a', 'b', 'c', 'd') default 'd'",
+			expectCapableOfInstantDDL: true,
+		},
+		{
+			name:                      "enum: fail insert in middle",
+			create:                    "create table t(id int, c1 enum('a', 'b', 'c'), primary key(id))",
+			alter:                     "alter table t modify column c1 enum('a', 'b', 'x', 'c')",
+			expectCapableOfInstantDDL: false,
+		},
+		{
+			name:                      "enum: fail change",
+			create:                    "create table t(id int, c1 enum('a', 'b', 'c'), primary key(id))",
+			alter:                     "alter table t modify column c1 enum('a', 'x', 'c')",
+			expectCapableOfInstantDDL: false,
+		},
+		{
+			name:                      "set: append",
+			create:                    "create table t(id int, c1 set('a', 'b', 'c'), primary key(id))",
+			alter:                     "alter table t modify column c1 set('a', 'b', 'c', 'd')",
+			expectCapableOfInstantDDL: true,
+		},
+		{
+			name:                      "fail set append when over threshold", // (increase from 8 to 9 values => storage goes from 1 byte to 2 bytes)
+			create:                    "create table t(id int, c1 set('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h'), primary key(id))",
+			alter:                     "alter table t modify column c1 set('a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i')",
+			expectCapableOfInstantDDL: false,
+		},
+	}
+	for _, tcase := range tcases {
+		t.Run(tcase.name, func(t *testing.T) {
+			if tcase.capableOf == nil {
+				tcase.capableOf = capableOf
+			}
+			createTable, err := parser.Parse(tcase.create)
+			require.NoError(t, err)
```

Comment:
FWIW, I've found it helpful to include more info when there are failures. For example:
```
require.NoError(t, err, "failed to parse a CREATE TABLE statement from %q: %v", tcase.create, err)
```

Reply:
Fixed, and also: no need to add the `err` value, because the value of `err` is already being printed by `require.NoError`.